plugins/stable/_npe2_writers_guide.md

```diff
@@ -31,7 +31,7 @@ by the corresponding writer contribution in the manifest).
    **attributes**, (e.g. `{'name': 'My Layer', 'opacity': 0.6}`)
 
    ```python
-   def single_layer_writer(path: str, data: Any, attributes: dict) -> Optional[str]:
+   def single_layer_writer(path: str, data: Any, attributes: dict) -> List[str]:
```

Comment:
I'm not sure about this one. The example below seems to indicate that a single layer writer that successfully writes should return a list. In my own plugin, I tried returning a list with one string, just the string, and `None`, and they all work. So maybe this should be `Union[List[str], str, None]`?

Reply:
This is definitely confusing. The part you highlighted disagrees with the type definition quoted just below this part.  In practice either works.

The intent was that line 34 is correct as is, so the type definition should get updated in the guide. This guide is generated from a [doc template in npe2](https://github.com/napari/npe2/blob/0cb4f0330d57425adb159345b3404844589820cd/_docs/templates/_npe2_writers_guide.md.jinja). So it needs to be corrected over there.